tests/bluetooth/tester/src/l2cap.c

```diff
@@ -404,6 +423,31 @@ static void listen(uint8_t *data, uint16_t len)
 		   BTP_STATUS_FAILED);
 }
 
+static void credits(uint8_t *data, uint16_t len)
+{
+	const struct l2cap_credits_cmd *cmd = (void *) data;
+	struct channel *chan = &channels[cmd->chan_id];
+
+	if (!chan->in_use) {
+		goto fail;
+	}
+
+	if (chan->nbuf && bt_l2cap_chan_recv_complete(&chan->le.chan,
+							chan->nbuf) < 0) {
```

Comment:
```suggestion
	if (chan->nbuf && bt_l2cap_chan_recv_complete(&chan->le.chan,
						      chan->nbuf) < 0) {
```

Reply:
will fix those,  but wonder why it wasn't catch by CI..

Reply:
CI doesn't ensure that the indentation is "correct" only that tabs comes before spaces and never after.